components/Feature/TopicExplorer/index.js

```diff
@@ -26,10 +39,10 @@ const TopicExplorer = (props) => {
         <input
           type="text"
           className="govuk-input govuk-input--width-20"
-          placeholder="e.g. food, mental health, lockdown"
           value={searchTerm}
           onChange={onSearchChange}
         />
+        <div id='example-search'>Try a search for <span data-search-term='food' data-testid='food' onClick={populateInput} className='inlineLink'>food</span>, <span className='inlineLink' data-search-term='mental health' onClick={populateInput} >mental health</span>, or <span className='inlineLink' data-search-term='debt'onClick={populateInput} >debt</span></div>
```

Comment:
This long line is a little hard to read. It might be worth extracting the link into a separate method (I think you can do that without building a whole new component?)

Reply:
I have now split this into multiple lines to make it more readable